_posts/2016-11-21-recurse-center-week-8.md

```diff
@@ -0,0 +1,127 @@
+
+With my time at RC quickly running out, I've been trying to focus on Haskell so I can accomplish that goal. This has also meant ignoring all other ambitions. Hard to tell if one project is more efficient. It prevents context switches, but also provides no opportunity to work on other things when you feel fatigued. Might need to throw some more variety into the mix.
+
+Haskell is also one of the more niche languages at RC, so it can be hard to find in person help with it.
+
+#### **Hack the Stacks**
+
+Over the weekend I attended a Hackathon at the Museum of Natural History. It involved working on projects proposed by the museum's library department.
+
+There was a project I was initially excited about, that involved creating an API hub for the library. Unfortunately, the others interested in the project seemed very opinionated and disinterested in discussion. I just wanted to write code and have fun, not be micro-managed. So quietly snuck away from that group and back to the other RCers.
+
+Our project was to reassemble Charles Darwin's cut up notebook. Kind of like writing software to assemble a jig-saw puzzle.
+
+I focused on building a backend to store all the information and allow users to upvote/downvote matches. You can see the code [here](https://github.com/HackTheStacks/darwin-viewer).
+
+It was a fun project, and we ended up winning in the "Why didn't I think of that!" category 😎.
+
+### Physical 🛌
+
+#### **Night Owling**
+
+As a product of dropping out of checkins, I've been experimenting with being a night owl. In other words, I've been sleeping in until ~12pm, arriving ~2pm, staying until ~3am, going to sleep ~5am.
+
+**Pros:**
+- Everything is calmer, so it's easier to focus.
+- I sleep great and I'm not forcing myself out of the house at any specific time.
+- My time for pairing/collaboration is very time boxed. I usually have ~3 hours at the end of the day for helping others. After that, people tend to leave and I focus on my own things.
+
+**Cons:**
+- I'm extra disconnected from RC. I haven't gone for lunch with people in ages.
+- Evening activities fall right in the middle of my "work day" (eg. concerts, meetups outside of RC)
+- The metro runs very infrequently at night.
+
+It's fun to do for a week at a time though. I might switch to a more normal schedule in a bit...
+
+#### **Caffeine Addiction**
+
+Having already switched to a night schedule, coding 24 hours straight for the Hackathon seemed like a fun idea. Unfortunately, it became an idea fueled by large amounts of caffeine. I drank a ton of Club-Mate and coffee over the course of the night.
+
+The day after the Hackathon I decided to skip all coffee. By the evening I had a massive migraine and became very nauseous. Stupid caffeine!
+
+Going to stay off the coffee until the grossness passes...
+
+### </End>
+
+Feeling pretty good about the things. Really hoping to come up with a plan for post RC. If you have any thoughts, send them [my way](/contact)!
```

Comment:
Consider moving this sentence above to where you write about your thoughts post-RC.